scripts/dockerfiles/runtime/Dockerfile.debug-common

```diff
@@ -0,0 +1,22 @@
+ARG RUNTIME_IMAGE
+FROM ${RUNTIME_IMAGE}
+
+ENV S3_BUCKET ""
+ENV S3_KEY_PREFIX issue
+
+RUN yum install -y unzip zip gdb
+# Allow curl to skip if broken
```

Comment:
why?

Reply:
Previously this was one comamnd, https://github.com/aws/aws-for-fluent-bit/blob/1777caa1de71e501cf4b4e47d36a5ece2138e15d/scripts/dockerfiles/Dockerfile.main-debug-s3#L7, and when testing AL2023 there was a conflict with curl-minimal (installed in 2023 base image). I think I had separated this out as it seemed the right approach, but on retest it looks like:

`yum install -y unzip zip curl gdb --skip-broken`

Allows both AL2 and AL2023 to install/use existing dependencies so that `RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-$(arch).zip" -o "awscliv2.zip"` does not fail